pulp_smash/tests/pulp2/platform/cli/test_selinux.py

```diff
@@ -31,6 +31,12 @@
 """A single line of output from ``ps``."""
 
 
+def setUpModule():  # pylint:disable=invalid-name
+    """Skip tests if the either default requires or selinux are unavailable."""
+    set_up_module()
+    require_selinux()
```

Comment:
Let's say Pulp 2 is installed on Fedora, and for whatever reason, SELinux is disabled. In this case, this module of tests will skip, and neither the Pulp team nor the Pulp QE team will know about the lack of SELinux testing. Given the Pulp team's commitment to supporting SELinux, this omission is dangerous. This is especially true given the rich history of SELinux issues on Pulp 2.

If SELinux tests are to be skipped, please make sure to do so under narrowly defined circumstances. Would the sample code I've provided in https://github.com/PulpQE/pulp-smash/pull/975#issuecomment-384394011 work?

Reply:
Ah, I didn't see your latest comment regarding cgroups. I'll adjust the test to check for the container.